tests/rails-6_capybara-3/features/browser_tab_steps.feature

```diff
@@ -0,0 +1 @@
+../../shared/features/shared/browser_tab_steps.feature
```

Comment:
I thought all shared features are already symlinked through their hierarchy, isn't that so?